.env.development

```diff
@@ -7,3 +7,4 @@ REACT_APP_USE_DOPPLER_LEGACY_SIGNUP=false
 REACT_APP_USE_DOPPLER_LEGACY_FORGOTPASSWORD=false
 REACT_APP_SHOPIFY_URL= https://sfy.fromdoppler.com:4433
 REACT_APP_DOPPLER_API_URL= http://newapiqa.fromdoppler.net
+REACT_APP_REPORTS_URL= https://reportsint.fromdoppler.net:400
```

Comment:
Here I would leave doppler local configuration in case we need it. Just a thought 😄